test/testdata/cyclop.go

```diff
@@ -0,0 +1,28 @@
+// args: -Ecyclop
+package testdata
+
+func cyclopComplexFunc() { // ERROR "calculated cyclomatic complexity for function cyclopComplexFunc is 11, max is 10"
+	i := 1
```

Comment:
```suggestion
package testdata

import "math"

func cyclopComplexFunc() { // ERROR "calculated cyclomatic complexity for function cyclopComplexFunc is 11, max is 10"
	i := math.MaxInt8
```

Reply:
What's the benefit with using `MaxInt8` and pulling in `math` as opposed to just using 1?

Reply:
It's related to https://github.com/golangci/golangci-lint/pull/1733

Reply:
Ah, right! Remember reading that now that you point it out - thanks! 🎉